src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredMen = people.filter(person => (
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  ));
+
+  const averegeAge = filteredMen.reduce((prev, { died, born }) =>
+    (prev + (died - born)), 0) / filteredMen.length;
```

Comment:
```suggestion
  const averegeAge = filteredMen.reduce((prev, { died, born }) => (
    prev + (died - born)
  ), 0) / filteredMen.length;
```

it's easier to get where reduce ends, if you write it this way